opus/import/obs_cassini_common_pds3.py

```
@@ -0,0 +1,106 @@
+            target_name_info = config_targets.TARGET_NAME_INFO[target_desc]
+            return target_desc, target_name_info[2]
+
+        if target_name not in config_targets.TARGET_NAME_INFO:
+            self._announce_unknown_target_name(target_name)
+            if self._ignore_errors:
+                return 'None', None
+            return None, None
+        target_info = config_targets.TARGET_NAME_INFO[target_name]
+        return target_name, target_info[2]
+
+    def _fix_cassini_sclk(self, count):
```

Ruff (0.14.8)

Missing return type annotation for private function `_fix_cassini_sclk`

(ANN202)